libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.kt

```diff
@@ -531,4 +511,25 @@ open class PushService {
             }
         }
     }
+
+    enum class PushNotificationRegistrationType {
```

Comment:
Should the class name say ReRegistration ie be PushNotificationReRegistrationType?
Should the values "simply" be:
- ReRegistrationDisabled
- ReRegistrationOnAppForeground
- ReRegistrationPeriodically

Reply:
Updated!  Badda-bing, badda boom 🚀